src/test/java/com/epam/tc/steps/ActionSteps.java

```diff
@@ -0,0 +1,68 @@
+package com.epam.tc.steps;
+
+import com.epam.tc.Exceptions.NotFoundLabelFromBoardException;
+import com.epam.tc.data.DataToRemove;
+import com.epam.tc.dto.BoardDto;
+import com.epam.tc.dto.LabelDto;
+import com.epam.tc.dto.ListDto;
+import com.epam.tc.service.TrelloService;
+import com.epam.tc.tests.TrelloFunctionTests;
+import java.util.List;
+
+public class ActionSteps {
+
+    public static ActionSteps createBoard(String boardName) {
+        TrelloFunctionTests.boardDto = TrelloService.createBoardByName(boardName);
```

Comment:
Object from particular test class (TrelloFunctionTests.boardDto) is still used in steps level, please fix it